### ISSUE TITLE
Dashboard displaying empty data

### ISSUE THREAD

**Issue by gsabater** [swdotcom/swdc-vscode#3]
Hello,

Even with keystrokes recorded on the left panel, the right sidebar has no data for any of my activities.
Also, I can't find any list of the time spent by project anywhere, and I think that this is the most interesting feature of the project.

![nope](https://user-images.githubusercontent.com/4948172/51918163-5314da00-23e1-11e9-9a8c-ca24d840da2b.JPG)

Also, local dashboard reports 0 as well.

**Comment by brettmstevens7** [comment]
Hey thanks @gsabater for calling this to our attention. @rick-software is working on a fix for this now. 

Time by project will appear in your dashboard every Sunday with a summary of the previous week. We also just added a new command to see your dashboard by clicking on "Software.com" in your status bar:

![vs-code-dashboard](https://user-images.githubusercontent.com/27828739/52011709-9af13b00-248d-11e9-8739-3bee410add6d.gif)

**Comment by brettmstevens7** [comment]
Hi Gaspar, we just pushed v0.5.76 if you don't mind trying out your dashboard in VS Code. The metrics on the top right of your feed should be updating throughout the day, and the metrics below (Code time stats) should show data from the trailing 7, 14, and 30 days.

**Comment by gsabater** [comment]
Hello @brettmstevens7 
I used the version 0.5.76 yesterday, and today i find this on the dashboard:

![ddd](https://user-images.githubusercontent.com/4948172/52630952-e0fec380-2ebd-11e9-8da9-27633f826035.JPG)

* This image says i have 0 average hours, strokes and strokes per minute, while I do have data for yesterday, today and past days. Avergae should never be 0, even if the data comes from the same day

* I'm in Spain, here is already Feb 12, but the dashboard must be in another timezone where is still monday. An option to define timezone is important I think, or I might get scrambled and inaccurate data.

![top](https://user-images.githubusercontent.com/4948172/52631168-55396700-2ebe-11e9-95b8-010507b98c97.JPG)

* Personally, for me this is the most important panel. I would love to pin this panel to the top, and also I find it crucial to be able to filter per days or any other time range. I MUST know how much time I have inverted today in a project. And it would be also neat if I could see at what times I do have worked on them. For example, a client calls me and asks me to do something on their site. In the morning I spend 45min doing that, and while I send them an email to notify them, they ask me another task. If I'm 30 minutes with that, is cool to see 1h15min, but also, is cool to see 45min at 10:00 and 30 min at 15:00, because i might bill one task and not the other one.
